### PR TITLE
feat: navigation

### PR DESCRIPTION
**Pull request by harshitadaswani**
added: navigation simple desktop

### CODE REVIEW

**Review by dev-enforced — APPROVED** [review]
The code looks good to me !!Use of the variables and naming conventions is awesome👍🏻!! You can merge it

File: components/navigation.html [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+                    </div>
+                    <div class="nav-right">
+                        <a href="https://github.com/harshitadaswani" target="_blank" rel="noopener noreferrer" class="txt-l fw-black p-s m-s"><i class="fab fa-github-square icons"></i></i></a>
+                        <a href="https://www.linkedin.com/in/harshitabdaswani/" target="_blank" rel="noopener noreferrer" class="txt-l fw-black p-s m-s"><i class="fab fa-linkedin icons"></i></a>
+                        <a href="https://twitter.com/harshi_daswani" target="_blank" rel="noopener noreferrer" class="txt-l fw-black p-s m-s"><i class="fab fa-twitter-square icons"></i></a>
+                    </div>
+                </nav>
+            </div> 
+            
+            <iframe src="https://www.thiscodeworks.com/embed/620023e64dd44f0015f3cca5" style="width: 100%; height: 20rem;" class="code-snippet border-bottom" frameborder="0"></iframe>
```
Comment (rkapoor10) [inline components/navigation.html:74]: read about html code tag.

Comment (rkapoor10) [reply components/navigation.html:74]: and you can use it here

**Review by rkapoor10 — APPROVED** [review]
Good to go. Merge it.

File: components/navigation.html (hunk 1 above)
Comment (harshitadaswani) [reply components/navigation.html:74]: will see to it

**Pull request merged**